src/includes/includer/debug.rs

```diff
@@ -20,9 +20,16 @@
 
 use super::prelude::*;
 use crate::tree::VariableMap;
+// Imported so that tests work, otherwise not used
+#[allow(unused_imports)]
```

Comment:
If an import is used only for tests, then it should only be included for tests:
```suggestion
#[cfg(test)]
```